src/firefly/js/ui/ImageSelect.jsx

```diff
@@ -320,6 +323,12 @@ function Header({project, multiSelect}) {
                 labelWidth={35}
                 wrapperStyle={{whiteSpace: 'normal'}}
             />
+            </div>
+            <div style={{marginLeft: -5}}><div>
+                <a target='_blank' href={href}>
+                    <img
+                     src={infoIcon}/></a>
+            </div></div>
```

Comment:
You may want to check to see if `href` has a value before rendering.  I assume there may be some dataset without info link.